pkg/ec2helper/ec2helper.go

```diff
@@ -1475,7 +1473,7 @@ func (h *EC2Helper) LaunchFleet(templateId *string) (*ec2.CreateFleetOutput, err
 	}
 
 	spotRequest := &ec2.SpotOptionsRequest{
-		AllocationStrategy: aws.String("capacity-optimized"),
+		AllocationStrategy: aws.String("capacity-optimized-prioritized"),
```

Comment:
Why did this change? The line on the main repo is just `capacity-optimized`.

https://github.com/awslabs/aws-simple-ec2-cli/blob/main/pkg/ec2helper/ec2helper.go#L1456

Reply:
Reverted